src/Compilers/CSharp/Test/Emit/CodeGen/PatternTests.cs

```diff
@@ -31,16 +31,7 @@ static class C {
             compilation.GetDiagnostics().Verify();
             compilation.GetEmitDiagnostics().Verify(
                 // warning CS8021: No value for RuntimeMetadataVersion found. No assembly containing System.Object was found nor was a value for RuntimeMetadataVersion specified through options.
-                Diagnostic(ErrorCode.WRN_NoRuntimeMetadataVersion).WithLocation(1, 1),
-                // (9,48): error CS0518: Predefined type 'System.Nullable`1' is not defined or imported
-                //     public static bool M() => ((object)123) is int i;
-                Diagnostic(ErrorCode.ERR_PredefinedTypeNotFound, "int i").WithArguments("System.Nullable`1").WithLocation(9, 48),
-                // (9,48): error CS0656: Missing compiler required member 'System.Nullable`1.GetValueOrDefault'
```

Comment:
We aren't hitting them because those code paths don't exist any longer. We use a different lowering strategy for this code now.